### DISCUSSION TITLE
iStoreOS translate

### DISCUSSION THREAD

**Discussion by markrieder** [istoreos/istoreos#1854] [Ideas]
Hello, we want to help with iStoreOS translation. 

https://crowdin.com/

**Comment by fejeIaqbhbv** [comment]
Is there anything that hasn't been translated?

If you don't have Tencent QQ, try contacting the administrator on Telegram.
https://github.com/istoreos#%E5%85%B3%E6%B3%A8%E6%9B%B4%E6%96%B0

**Comment by markrieder** [comment]
The entire OS is in Chinese and is unusable for the rest of the world.

Dňa pi 18. 10. 2024, 3:11 fejeIaqbhbv ***@***.***> napísal(a):

**Reply by fejeIaqbhbv** [reply]
Are you serious?

This is based on OpenWrt!

If you say that a few features haven't been translated, I can understand that, but `The entire OS is in Chinese` is simply impossible.

OpenWrt's LuCI will automatically recognize your browser information; non-Chinese browsers will default to English when opening LuCI, unless you forcibly lock the language to Chinese.

![p](https://github.com/user-attachments/assets/20d85c1b-0e57-499a-9af8-b9abab5c35bb)
